hoomd/md/test/test_nve_integrator.cc

```diff
@@ -420,8 +421,9 @@ void nve_updater_aniso_test(std::shared_ptr<ExecutionConfiguration> exec_conf,
     std::shared_ptr<NeighborList> nlist_1(new NeighborListBinned(sysdef_1, r_cut, r_buff));
 
     nlist_1->setStorageMode(NeighborList::full);
-    std::shared_ptr<AnisoPotentialPairGB> fc_1
-        = std::shared_ptr<AnisoPotentialPairGB>(new AnisoPotentialPairGB(sysdef_1, nlist_1));
+    std::shared_ptr<AnisoPotentialPair<EvaluatorPairGB>> fc_1
+        = std::shared_ptr<AnisoPotentialPairGB>(
+            new AnisoPotentialPair<EvaluatorPairGB>(sysdef_1, nlist_1));
```

Comment:
```suggestion
    auto fc_1 = std::make_shared<AnisoPotentialPair<EvaluatorPairGB>>(sysdef_1, nlist_1));
```
While we're fixing this line, let's just make it a lot less verbose.